packages/grafana-ui/src/components/Table/TableCellInspector.tsx

```diff
@@ -28,11 +29,17 @@ export function TableCellInspector({ value, onDismiss, mode }: TableCellInspecto
     if (trimmedValue[0] === '{' || trimmedValue[0] === '[' || mode === 'code') {
       try {
         value = JSON.parse(value);
-        displayValue = JSON.stringify(value, null, '');
-      } catch {}
+        displayValue = JSON.stringify(value, null, '  ');
```

Comment:
Yay! I'm excited this bug is being addressed 💯 great job Kyle! 

I'm wondering if we want this to be one space instead of two? (Original was just one space)

Reply:
never seen source code that's indented just one space, but i'm sure it exists!

we use two in panel inspect, so probably consistency is good, too.

![image](https://github.com/user-attachments/assets/b6a13150-a831-422c-b5c5-c9292cba0ccf)